datetimerange/__init__.py

```diff
@@ -708,4 +725,4 @@ def __validate_value(self, data_prop):
 
     @staticmethod
     def __get_timedelta_sec(dt):
-        return int(dt.days * 60 ** 2 * 24 + float(dt.seconds) + dt.microseconds / (1000.0 ** 2))
+        return int(dt.days * 60 ** 2 * 24 + float(dt.seconds) + dt.microseconds / (1000.0 ** 2))
```

Comment:
line missing at the end. Is this your editor?

Reply:
I think so